src/source/worker.js

```diff
@@ -132,20 +132,20 @@ export default class Worker {
             this.self.importScripts(params.url);
             callback();
         } catch (e) {
-            callback(e);
+            callback(e.toString());
```

Comment:
It's been a while since I wrote this, but yeah, what I remember is that some `Errors` that get generated deep in the `importScripts` code don't serialize.